static/templates/humanitarian.html

```diff
@@ -63,26 +63,25 @@ <h3 class="panel-title">Narrative</h3>
         </div>
         <div class="panel-body">
           <h5>Publisher</h5>
-        	<p>Only publishers with humanitarian activities included.</p>
+          <p>The name that this publisher self-defines as on the IATI Registry.</p>
 
           <h5>Publisher Type</h5>
           <p>The category that this publisher self-defines as on the IATI Registry.</p>
 
           <h5>Number of Activities</h5>
-          <p>Total number of humanitarian activities (determined by use of <code>iati-activity/@humanitarian</code> attribute or DAC 5-digit sector codes between <code>72010</code> to <code>74010</code> inclusive, or DAC 3-digit sector codes <code>720</code>, <code>730</code> or <code>740</code>).</p>
+          <p>Total number of humanitarian activities — determined by the presence of of either the <code>iati-activity/@humanitarian</code> attribute (set to <code>1</code> / <code>true</code>), at least one <code>iati-activity/transaction/@humanitarian</code> attribute (set to <code>1</code> / <code>true</code>) (both criteria apply if the activity is published at IATI version 2.02 and higher), or DAC 5-digit sector codes between <code>72010</code> to <code>74010</code> inclusive, or DAC 3-digit sector codes <code>720</code>, <code>730</code> or <code>740</code>) (for activities published at any IATI version).</p>
```

Comment:
> of of

^^ typo

Reply:
Good spot! Will fix these today and send in another PR :-)

Reply:
> between `72010` to `74010` inclusive

Mm… Maybe this is fine… But codelist codes are not ordered, so it’s a bit ambiguous to say “between”. I guess this is okay.

Reply:
In general, this is a very very long sentence. I wonder if it could be fragmented a bit?

One thing I would suggest is perhaps to remove the stuff about versions. I guess it’s useful to know that the humanitarian attribute can only be used at v2.02+, but that’s not specific to this methodology – rather, it’s because that was when the humanitarian attribute was added to the standard. So in that sense, I’d be tempted not to mention it here.

Reply:
These issues are fixed (or at least improved) in #533

Reply:
Confirmed, reviewed and merged